resources/public/api/conf/1.0/scenarios/retrieve_period_summary_scenarios.md

```diff
@@ -19,5 +19,9 @@
             <td><p>NOT_FOUND</p></td>
             <td><p>Simulates the scenario where no data is found.</p></td>
         </tr>
+        <tr>
+            <td><p>TAX_YEAR_NOT_SUPPORTED</p></td>
+            <td><p>Simulates the scenario where the tax year is not supported.</p></td>
+        </tr>
```

Comment:
This looks good. However I think this scenario also needs to be added to create_period_summary_scenarios.md as well(Spec can be found [here](https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?pageId=535069044) )